dotnet/samples/GettingStartedWithProcesses/Step04/Plugins/WeatherPlugin.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System.ComponentModel;
+using Microsoft.SemanticKernel;
+
+namespace Step04.Plugins;
+
+internal sealed record WeatherForecast(
+    string Date,
+    string Location,
+    string HighTemperature,
+    string LowTemperature,
+    string Precipition);
+
+/// <summary>
+/// Mock plug-in to provide weather information.
+/// </summary>
+internal sealed class WeatherPlugin
+{
+    private readonly Dictionary<string, WeatherForecast> _forecasts = [];
+
+    [KernelFunction]
+    public string GetCurrentDate() => DateTime.Now.Date.ToString("dd-MMM-yyyy");
```

Comment:
UTCNow instead? :)

Reply:
Wouldn't the fictional user be interested in their own locale time? ;)

Reply:
I always use UTC as default - otherwise bugs arise, but for a sample does not matter :)

Reply:
Agree.  UTC is good for service-side consistency.  User-locale is good for UX specific cases.